distributed/http/worker.py

```diff
@@ -17,9 +19,38 @@ def get(self):
         self.write(resp)
 
 
+class Data(RequestHandler):
+    """List of keys on this worker (scheduler also knows this)"""
+    def get(self):
+        self.write({'keys': list(self.server.data.keys())})
+
+
+class Value(RequestHandler):
+    """The value associated with a specific key.
+    Gives None (NULL) if not found."""
+    def get(self, key):
+        self.write({key: self.server.data.get(key, None)})
```

Comment:
The value will often be unserializable.  I'm sort of against exposing values through the JSON interface without a lot more care.